src/storage_helper_methods/storage.rb

```diff
@@ -0,0 +1,23 @@
+require 'json'
+
+module Storage
+  def update_data(array, file_path)
+    # File.new(file_path, 'w+') unless File.exist?(file_path)
```

Comment:
- [ ] Same goes here where **I suggest that you remove all the unnecessary commented-out codes like this one above.** This is among the best practices that every developer is supposed to follow 🏃